_docs/_latest/use/inventory.md

```diff
@@ -26,8 +29,8 @@ The command above will start a new inventory process, you can track the status b
 
 ### Create a new inventory along with a data model
 
-Since the data model is widely used in Forseti so sometimes it's handy 
-to just create the dada model along with the newest inventory.
+Since the data model is widely used in Forseti, sometimes it's handy 
```

Comment:
Simplify for globalization "Because the data model is widely used in Forseti, it can be helpful to create the data model along with the newest Inventory."

Reply:
done